fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -816,12 +817,21 @@ ACTOR Future<Void> commitBatch(
 		// Serialize the log range mutations within the map
 		for (; logRangeMutation != logRangeMutations.end(); ++logRangeMutation)
 		{
-			if(yieldBytes > SERVER_KNOBS->DESIRED_TOTAL_BYTES) {
-				yieldBytes = 0;
-				wait(yield());
+			valueWriter = BinaryWriter(IncludeVersion());
```

Comment:
It would be nice if we can keep MutationListRef::blob_begin private. If we resolve #2146 then we can also re-use the serialization code for both the asynchronous and synchronous versions. I think this is ok for now.

Reply:
I will add a FIXME near both serialize functions